content-repo/extra-docs/releases/20.7.0.md

```diff
@@ -0,0 +1,516 @@
+
+---
+
+### ServiceNow Pack v1.1.5
+#### Integrations
+##### ServiceNow v2  
+Fixed the test button to work with debug mode.
+
+---
+
+### Slack Pack v1.3.1
+#### Integrations
+##### SlackV2  
+Increased integration context reliability by using versions (supported in Cortex XSOAR v6.0 and later).
+
+---
+
+### Symantec Management Center Pack v1.0.1
+#### Integrations
+##### Symantec Management Center
+Added support for content type LOCAL_CATEGORY_DB.
+
+---
+
+### TAXII Feed Pack v1.0.1
+#### Integrations
+##### TAXII Feed  
+Added the *Tags* parameter.
+
+---
+
+### ThreatConnect Pack v2.0.1
+#### Integrations
+##### ThreatConnect  
+Deprecated. Use the ThreatConnect v2 integration instead
+
+##### ThreatConnect v2  
+Threat intelligence platform.
+
+---
+
+### ThreatQ Pack v1.0.3
+#### Integrations
+##### ThreatQ v2  
+Fixed an issue where the the URL schema was enforced in the ***url*** command.
+
+---
+
+### TruSTAR Pack v2.0.0
+#### Integrations
+##### TruSTAR v2  
+The TruSTAR v2 integration introduces rewritten code, tests, docstrings on code functions, and new commands.
+For commands that return indicators, the data is put in 3 contexts:
+- The standard context, without the malicious field because TruSTAR doesn't currently have a score for every indicator.
+- DBotScore context with score as 0 for the same reason.
+- TruSTAR context with all the information returned by the command.
+Commands:
+  - ***trustar-get-reports***
+  - ***trustar-get-enclaves***
+  - ***trustar-related-indicators***
+  - ***trustar-indicators-metadata***
+  - ***trustar-indicator-summaries***
+  - ***trustar-get-whitelisted-indicators***
+  - ***trustar-move-report***
+  - ***trustar-trending-indicators***
+  - ***trustar-get-indicators-for-report***
+  - ***trustar-search-indicators***
+  - ***trustar-submit-report***
+  - ***trustar-delete-report***
+  - ***trustar-correlated-reports***
+  - ***trustar-add-to-whitelist***
+  - ***trustar-remove-from-whitelist***
+  - ***trustar-report-details***
+  - ***trustar-update-report***
+  - ***trustar-search-reports***
+  - ***trustar-get-phishing-indicators***
+  - ***trustar-get-phishing-submissions***
+  - ***trustar-set-triage-status***
+  - ***trustar-copy-report***
+
+##### TruSTAR  
+Deprecated - Use the TruSTAR v2 integration instead.
+
+---
+
+### US - Breach Notification Pack v1.0.3
+#### Layout
+##### US Breach Notification  
+- Added a new widget to the layout.
+- Added Notifications status to the layout.
+
+#### Playbooks
+##### Illinois - Breach Notification  
+Added setincidents to the playbook for the new layout.
+
+##### New York - Breach Notification  
+Added setincidents to the playbook for the new layout.
+
+##### California - Breach Notification  
+Added setincidents to the playbook for the new layout.
+
+##### Residents Notification - Breach Notification  
+Added setincidents to the playbook for the new layout.
+
+---
+
+### Zoom Feed Pack v1.1.0
```

Comment:
I think this whole pack is new and needs to be moved to the top with the New stuff.